### ISSUE TITLE
Possible to get predictions on the "link" scale with predict.brmsfit?

### ISSUE THREAD

**Issue by atrolle** [paul-buerkner/brms#274]
Hi Paul,

Using the predict.brmsfit function, would it be possible to get predictions on the "link" scale (rather than on the "response" scale), just like some ML packages (e.g. vcrpart) do?

Thank you in advance,
Arnaud.

**Comment by paul-buerkner** [comment]
Use fitted (., scale = "linear")

Am 13.10.2017 09:31 schrieb "atrolle" <notifications@github.com>:

**Comment by atrolle** [comment]
Thank you.

Best regards,
Arnaud.

**Comment by atrolle** [comment]
Hi again,

I'm not sure to understand the third dimension ("P(Y = 1)","P(Y = 2)", "P(Y = 3)") of the output array obtained from the fitted function. I ran an ordinal regression analysis using "acat" family, and my ordinal DV has 4 levels.

Could you enlighten me ?

Best,
Arnaud.

**Comment by paul-buerkner** [comment]
I would expect to see the third dimension having length 4 (because you have 4 categories). Are you sure you have 4 levels and could you show me a reproducible example?

**Comment by atrolle** [comment]
Ok, I've got it.
I may have found the answer in a course : when using adjacent logits, K-1 logit equations are computed, with K categories. That would be why I get three elements "P(Y = 1)","P(Y = 2)", and "P(Y = 3)" along the 3rd dimension of the output.
Thank you again,
Arnaud.

**Comment by paul-buerkner** [comment]
No, this is not the answer. As the names say ("P(Y = 1)" etc.) those are probabilities so if you have 4 categories, you should get a 3rd dimension of length 4. Again, are you sure you really have 4 categories?

**Comment by atrolle** [comment]
Yes, I checked; I really have four categories.
I'm a bit lost. If I use the option "scale = linear" for the "fitted" function, and as there are three logit equations, shouldn't I get three predicted values along the 3rd dimension of the output?

**Comment by paul-buerkner** [comment]
Oh you are right, sorry. I forgot that you used scale = "linear"...

**Comment by paul-buerkner** [comment]
I think I should change the names of the dimensions in this case.

**Comment by atrolle** [comment]
Ok. Fine, you reassure me :-).

Thanks,
Arnaud.

**Comment by paul-buerkner** [comment]
The problem is now fixed in the dev version of brms.
